integr8ly/managed-service-broker
# mdc service

## Motivation


https://issues.jboss.org/browse/AEROGEAR-9700

## What


Adding MDC as a new service to be managed.

## Why


So it can be accessed through the solution explorer ui

## How


Adding a new service package.

## Verification Steps


* Build an image using this pr or use mine: `docker.io/odranoel/managed-service-broker:master`
* Change msn deployment to use the custom dev image
* Set the following env var to a url: `MDC_DASHBOARD_URL`
* Wait for the pod to be ready and check the logs - there shoild be a log messafe wrt mdc

NOTE: can be used together with https://github.com/integr8ly/tutorial-web-app/pull/494 in case you are feeling adventurous
## Checklist:


## Progress


## Additional Notes
